pipelines/_ingredients/ApPipe.yaml

```diff
@@ -19,6 +19,8 @@ parameters:
   coaddName: goodSeeing
   # APDB config file must be user-specified
   apdb_config:
+  # Use release_id=0 for development, 1 for production, and 2 or 3 for reprocessing production data
+  release_id: 1
```

Comment:
If production is the default, then Prompt Processing doesn't need to do anything and DM-52170 is a no-op. However, _do_ we want production to be the default? Dev environments tend to be less tightly controlled, so it's more likely that someone would forget to override there.

Reply:
I recommend overriding `release_id=0` in `ap_verify` regardless of the above.

Reply:
Thank you for catching this! I meant to use `release_id: 0`, but accidentally committed an edit I used for testing. I'll change that, and set the parameter explicitly in `ap_verify` as you suggest.